src/docbkx/chapter77-express-data-prep.xml

```diff
@@ -0,0 +1,370 @@
+            </mediaobject>
+
+            Then fill in the following form, test the connection and save it.
+            <mediaobject>
+                <imageobject>
+                    <imagedata fileref="edp-monitor-setup-dialog.png" format="PNG" scalefit="1"/>
+                </imageobject>
+            </mediaobject>
+        </para>
+    </section>
+
+    <section id="edp_wizard_datastores">
+        <title>Datastores</title>
+        <emphasis>Supported formats</emphasis>
+        <para>
+            Express Data Prep Wizard supports CSV (comma separated value) and Excel data files. You need to provide
+            one within the wizard so that desired format (header) can be recognized. No real data is
+            required to be in this file. Just column names.
+            <mediaobject>
+                <imageobject>
+                    <imagedata fileref="edp-wizard-datastore-page.png" format="PNG" scalefit="1"/>
+                </imageobject>
+            </mediaobject>
+        </para>
+        <para>
+            Later on, when you upload your data files to the incoming hot folder, all those files have to follow
+            exactly the same format (header). Otherwise the job execution fails.
+        </para>
+        <para>
+            See <link linkend="edp_wizard_input_output">Input&amp;output</link> section for more details.
+        </para>
+    </section>
+
+    <section id="edp_wizard_input_output">
+        <title>Input&amp;output</title>
+        <emphasis>Incoming hot folder</emphasis>
+        <para>
+            In the final phase of the wizard, when you decide to upload the job to DataCleaner Monitor, you can specify
+            a directory that will be treated as <emphasis>incoming hot folder</emphasis>. This directory will be then
+            regularly scanned for changes. Whenever such a change occurs, new/changed file will be considered as
+            a new data input. That will trigger the job execution and process the data.
+            <mediaobject>
+                <imageobject>
+                    <imagedata fileref="edp-wizard-upload.png" format="PNG" scalefit="1"/>
+                </imageobject>
+            </mediaobject>
+        </para>
+        <para>
+            It is important to keep the file format (header) that is required by the job. Incorrect
+            file format will be recognized and reported within a failed job execution.
+        </para>
+
+        <emphasis>Outgoing folders</emphasis>
+        <para>
+            On the <link linkend="service_settings_page">Service settings page</link> you can specify
+            <emphasis>outgoing folder path</emphasis> and <emphasis>outgoing presorting reports path</emphasis>. Both
+            of these directories will serve as target directory where final output files will be put.
+            <mediaobject>
+                <imageobject>
+                    <imagedata fileref="edp-wizard-services-page.png" format="PNG" scalefit="1"/>
```

Comment:
I don't understand. 

Instead of: 
![01](https://user-images.githubusercontent.com/13213915/27033120-35d48c22-4f78-11e7-8ffe-fa422228ce56.png)

You want: 
![02](https://user-images.githubusercontent.com/13213915/27033122-38157a82-4f78-11e7-98b9-7c87d1896b33.png)

?